VMEncryption/main/DiskUtil.py

```diff
@@ -254,6 +293,9 @@ def export_token(self,device_name):
         keyEncryptionKeyUrl=disk_encryption_setting[CommonVariables.KeyEncryptionKeyURLKey]
         wrappedProtector = disk_encryption_setting[CommonVariables.PassphraseKey]
         attestationUrl = disk_encryption_setting[CommonVariables.AttestationURLKey]
+        if disk_encryption_setting[CommonVariables.PassphraseNameKey] != CommonVariables.PassphraseNameValueProtected:
+            self.logger.log("passphrase is not Protectected. No need to do SKR.")
```

Comment:
Under what circumstances do we store the not protected passphrase in the LUKS header. I'd expect this never to happen.

Reply:
There are no circumstances, in which we are keeping unprotected passphrase in LUKS header. 
we are always stamping PassphraseNameKey attribute to CommonVariables.PassphraseNameValueProtected in LUKS2 header, 
if by any chance, this value is changed than CommonVariables.PassphraseNameValueProtected, the code path must not got for SKR. 
Do you see this as warning?